rest/data/task.go

```diff
@@ -203,7 +112,7 @@ func (tc *DBTaskConnector) SetTaskActivated(taskId, user string, activated bool)
 
 // ResetTask sets the task to be in an unexecuted state and prepares it to be run again.
 // If given an execution task, marks the display task for reset.
-func (tc *DBTaskConnector) ResetTask(taskId, username string) error {
+func ResetTask(taskId, username string) error {
```

Comment:
Can we get rid of this function and either move this logic directly into the route that uses it or make it a helper function for the route?